packages/models-library/tests/test_service_settings_labels.py

```diff
@@ -130,3 +130,13 @@ def test_simcore_services_labels_compose_spec_null_container_http_entry_provided
     sample_data["simcore.service.compose-spec"] = None
     with pytest.raises(ValidationError):
         SimcoreServiceLabels(**sample_data)
+
+
+def test_raises_error_wrong_restart_policy() -> None:
+    simcore_service_labels: Dict[str, Any] = deepcopy(
+        SimcoreServiceLabels.Config.schema_extra["examples"][2]
+    )
+    simcore_service_labels["simcore.service.restart-policy"] = "__not_a_valid_policy__"
+
+    with pytest.raises(ValueError):
+        SimcoreServiceLabels(**simcore_service_labels)
```

Comment:
could you also bring that in the service integration tool please?

Reply:
Yes.